index.d.ts

```diff
@@ -0,0 +1,9 @@
+type ClassValue = string | number | ClassDictionary | ClassArray;
+
+interface ClassDictionary {
+	[id: string]: boolean;
+}
+
+interface ClassArray extends Array<ClassValue> { }
+
+export default function(...classes: ClassValue[]): string;
```

Comment:
Ah, you don't need that part. That was a remnant from before we had `@types` on NPM -- when you use those typings through `tsd` or the `typings` tool, you would need the ambient declaration of the `"classnames"` module. Now that we have node-style resolution of index.d.ts, that part isn't needed.